Scenes/Screens/RevivalShoop/RevivalShoop.gd

```diff
@@ -0,0 +1,42 @@
+extends Node2D
+
+onready var game_manager = get_parent().get_node("GameManager")
+
+const ENEMIES_PER_LEVEL_PRICE = 0
+const EARN_GOLD_PRIVE = 0
+
+func _ready():
+	game_manager.stop_gameplay()
+
+func _process(delta):
+	set_enemies_per_level_button()
+
+###########
```

Comment:
Do we really need that comment? This is probably the only one file with that, so it's little weird, isn't it?

Reply:
yes, but these pairs (three each) will be a lot, hmmm I can give two empty lines between

Reply:
It's ok for me. If you think, that's helpful, just leve this as is. I asked about it because it was something new in this project. Let's keep code as clean as we can ;)